### DISCUSSION TITLE
Two Smart meters IR sensor at one Wemos D1 mini

### DISCUSSION THREAD

**Discussion by Marty56** [arendst/Tasmota#15602] [Q&A] [answered]
Hi,

I have a Wemos D1 mini and have a smart meter IR sensor (bidirectional SML protocol) connect to GPIO 3.
This is working fine.

Is is possible to connect a second smart meter IR sensor to this very same Wemos D1 mini and which GPIO could I use then.

regards Marty

**Comment by gemu2015** [comment, marked as answer]
yes you may connect several meters to one device. check examples in tasmota SML docs
you may use any GPIO  which is not tied to a booting function and also avoid GPIO 16.
on a generic Wemos you may use e.g. GPIO 4.

**Comment by Marty56** [comment]
Thanks for the answer.

**Comment by Marty56** [comment]
It is working!

I have only connected the RX Pin of my Smartmeter IR device to a GPIO because I thought that only one GPIO Pin is specified in the Script. It this correct? So the TX Pin is not connected.

The second meter is principally working but only receives its value very 4 minutes. Whereas the other meter is updated very 2 seconds.

So state differently does the Smart Meter Module use the TX Pin?
Should I connect it to a fixed voltage, e.g. GDN oder VDD?
